testing/src/main/java/org/oppia/android/testing/platformparameter/TestPlatformParameterConstants.kt

```diff
@@ -0,0 +1,25 @@
+package org.oppia.android.testing.platformparameter
+
+import javax.inject.Qualifier
+
+// These constants are only used for testing purpose.
```

Comment:
I suggest instead splitting these into separate files for each annotation, and pairing the name/default/server value with those files. You can then add KDocs for each one to explain what that test parameter is.

Reply:
I am confused about this suggested change. Can you explain what is the need to split the annotations into a separate file from the constants. 
Also these are just Test constants so I guess it will be okay to put them in one file and we can follow this suggestion with the Prod Constants.

Reply:
Each annotation usually is its own file (ex: https://github.com/oppia/oppia-android/blob/develop/domain/src/main/java/org/oppia/android/domain/feedbackreporting/ReportSchemaVersion.kt)
That said, it also makes sense to me to keep them altogether in this file as they're only test constants but I'll defer to Ben on his thoughts on this.

Reply:
Yeah understood now. I will include this thing in the Wiki (as a step while creating a Platform Parameter). Other than that I also prefer to keep test constants in one file.

Reply:
What benefit do we gain from keeping them in one file? I think we can evaluate the merits of both approaches.

I lean toward the convention of keeping files as specific as possible (oftentimes having overly generic packages/classes/utilities can result in them growing unboundedly due to not having a clear idea what should go into that component; maintaining specific files helps keep things focused).

I also find the current name confusing. This file really contains annotations and constants, not just contains. Either we should rename it, or keep the annotations and constants in 2 separate files. If we do the latter, I suspect organizing the annotations as individual files with their constants included will be clearer and easier to find when searching for code vs. the alternative considerations. This is another benefit from the per-file convention: it makes code discovery easier.

Reply:
Yeah this explaination helps us to understand the need to have separate files. Including this change